docs/topics/external-cloud-provider.md

```diff
@@ -0,0 +1,29 @@
+# External Cloud Provider
+
+To deploy a cluster using [external cloud provider](https://github.com/kubernetes-sigs/cloud-provider-azure), create a cluster configuration with external cloud provider template. For example,
+
+```bash
+clusterctl config cluster ${CLUSTER_NAME} --from templates/cluster-template-external-cloud-provider.yaml --kubernetes-version v1.17.3 --control-plane-machine-count=3 --worker-machine-count=3 > ${CLUSTER_NAME}.yaml
+```
+
+After control plane is up and running, deploy external cloud provider components (cloud-controller-manager and cloud-node-manager) using:
+
+```bash
+kubectl --kubeconfig=./${CLUSTER_NAME}.kubeconfig \
+  apply -f templates/addons/cloud-controller-manager.yaml
+```
+
+```bash
+kubectl --kubeconfig=./${CLUSTER_NAME}.kubeconfig \
+  apply -f templates/addons/cloud-node-manager.yaml
+```
+
+After components are deployed, you should see following pods in Running state:
+
+```bash
+kube-system   cloud-controller-manager                                            1/1     Running   0          31m
```

Comment:
nit: let's cheat a bit and change the age in the example so it doesn't seem like there has to be a delay between to the two in case anyone is paying close attention to the numbers
```suggestion
kube-system   cloud-controller-manager                                            1/1     Running   0          41s
```